### PR TITLE
handles tossing of units in bio book in import process

### PR DESCRIPTION
**Pull request by jpslav**
* Removes book units during import
* Makes unnecessary the `chapter_section` code in `OpenStax::Cnx::V1::[Page,BookPart]`
* Removes other unused code in `OpenStax::Cnx::V1::[Page,BookPart]`

### CODE REVIEW

**Comment by jpslav** [comment]
@Dantemss, @kevinburleigh75 can you review?

File: lib/tasks/demo/biology.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ assignments:
 
   - type: reading
     step_types: [ r, r, e, r, r, e, r, r, e, r, r, e, r, e, r, p, e, e, r, r, e ]
-    chapter_sections: [[1, 1, 0], [1, 1, 1], [1,1,2]]
+    chapter_sections: [[1, 0], [1, 1], [1, 2]]
     title: Read Unit 1. The Chemistry of Life
```
Comment (Dantemss) [inline lib/tasks/demo/biology.yml]: This is `Chapter 1. The Study of Life`

File: lib/tasks/demo/biology.yml [hunk 2]
```diff
@@ -40,7 +40,7 @@ assignments:
 
   - type: reading
     step_types: [ r, r, r, r, r, r, r, e, r, r, r, p ]
-    chapter_sections: [[2, 3, 0], [2, 3, 1], [2, 3, 2]]
+    chapter_sections: [[3, 0], [3, 1], [3, 2]]
```
Comment (Dantemss) [inline lib/tasks/demo/biology.yml]: This should be chapter 6, unless you really meant to change which chapter we're using.
Unit 2 chapter 3 becomes chapter 6.

File: lib/tasks/demo/biology.yml [hunk 3]
```diff
@@ -40,7 +40,7 @@ assignments:
...
     title: Read Unit 2. The Cell
```
Comment (Dantemss) [inline lib/tasks/demo/biology.yml]: This is either `Chapter 3. Biological Macromolecules` or `Chapter 6. Metabolism` depending on what you choose above.

File: lib/tasks/demo/biology.yml [hunk 4]
```diff
@@ -72,7 +72,7 @@ assignments:
 
   - type: homework
     step_types: [ e, e, e, e, e, e, e, e, e, e, p ]
-    chapter_sections: [[1, 1, 0], [1, 1, 1], [1,1,2]]
+    chapter_sections: [[1, 0], [1, 1], [1, 2]]
     title: HW The Chemistry of Life
```
Comment (Dantemss) [inline lib/tasks/demo/biology.yml]: `The Study of Life`

File: spec/fixtures/demo-imports/biology.yml [hunk 5]
```diff
@@ -14,7 +14,7 @@ assignments:
 
   - type: reading
     step_types: [ r, r, e, r, r, e, r, r, e, r, r, e, r, e, r, p, e, e, r, r, e ]
-    chapter_sections: [[1, 1, 0], [1, 1, 1], [1,1,2]]
+    chapter_sections: [[1, 0], [1, 1], [1, 2]]
     title: Read Unit 1. The Chemistry of Life
```
Comment (Dantemss) [inline spec/fixtures/demo-imports/biology.yml]: Same comments as the full demo.

File: spec/fixtures/demo-imports/biology.yml [hunk 6]
```diff
@@ -28,7 +28,7 @@ assignments:
 
   - type: homework
     step_types: [ e, e, e, e, e, e, e, e, e, e, p ]
-    chapter_sections: [[1, 1, 0], [1, 1, 1], [1,1,2]]
+    chapter_sections: [[1, 0], [1, 1], [1, 2]]
     title: HW The Chemistry of Life
```
Comment (Dantemss) [inline spec/fixtures/demo-imports/biology.yml]: Here too.

File: spec/subsystems/content/import_book_spec.rb [hunk 7]
```diff
@@ -55,4 +56,36 @@ def test_book_part(book_part)
...
+    part = root_book_part.child_book_parts.third
+    expect(part.title).to eq "Biological Macromolecules"
+    expect(part.chapter_section).to eq [3]
+
+    # The second page of that chapter
+
+    page = root_book_part.child_book_parts.third.pages.second
+    expect(page.title).to eq "Macromolecules"
+    expect(page.chapter_section).to eq [3,1]
+  end
```
Comment (Dantemss) [inline spec/subsystems/content/import_book_spec.rb]: Might wanna test that the numbering is correct for stuff that was in Unit 2.

Comment (jpslav) [inline spec/subsystems/content/import_book_spec.rb]: You r smrt.  Totally wasn't working.  Fix coming now.

**Pull request merged**